src/lightkurve/interact.py

```diff
@@ -350,6 +354,10 @@ def _add_nearby_tics_if_tess(tpf, source, tooltips):
     if tic_id is None or tic_id == "":
         return source, tooltips
 
+    if isinstance(tic_id, str):
+        # for cases tpf is from tpf.cutout() call in #1089
+        tic_id = tic_id.replace("_CUTOUT", "")
```

Comment:
The fix for tpf.cutout() is a bit hacky (by removing _CUTOUT suffix from the header value).

I presume there is some reason for _CUTOUT to be appended to TICID header in the first place, and opt for compensating for the special case here.